### PR TITLE
require.resolve('source-map-support') if using lerna

### LINKED ISSUE #42: Cannot find module Source-map-support
Hi, I'm trying to push my build on production and I'm getting this error

`Cannot find module '/Users/NAME/FOLDER/PROJECT_NAME/node_modules/source-map-support/source-map-support.js'`

Anyone know how to fix this?

Thank you very much!

### PR DESCRIPTION
**Pull request by ericclemmons**
Fixes #42.

I couldn't do `hub pull-request -i 42`:
> Error creating pull request: Unprocessable Entity (HTTP 422)
> Not allowed to change field "issue"

### CODE REVIEW

**Comment by jaredpalmer** [comment]
nice.

File: packages/backpack-core/config/webpack.config.js [hunk 1]
```diff
@@ -115,7 +115,14 @@ module.exports = (options) => {
       new webpack.BannerPlugin({
         raw: true,
         entryOnly: false,
-        banner: `require('${require.resolve('source-map-support/register')}')`
+        banner: `require('${
+            // Is source-map-support installed as project dependency, or linked?
+            require.resolve('source-map-support').match(process.cwd())
```
Comment (sompylasar) [inline packages/backpack-core/config/webpack.config.js]: `.indexOf(process.cwd()) === 0` would be more reliable than `.match(process.cwd())` if `.match` here is `String.prototype.match`, not anything special.

Comment (ericclemmons) [inline packages/backpack-core/config/webpack.config.js]: @sompylasar Can you clarify again?  I considered `indexOf`, but found `String.prototype.match` to read better.

Thanks!

Comment (sompylasar) [inline packages/backpack-core/config/webpack.config.js]: `match` finds in any place of the string, but `indexOf() === 0` ensures it's at the start.

Comment (ericclemmons) [inline packages/backpack-core/config/webpack.config.js]: Oh, of course!  I figured since it's `/Users/Me/whatever` the risk was low.

Comment (jaredpalmer) [inline packages/backpack-core/config/webpack.config.js]: @eric should we fix this on master with a new PR?

Comment (sompylasar) [inline packages/backpack-core/config/webpack.config.js]: > the risk was low.

Yes. Low but not zero. The underlying intended logic is `indexOf() === 0` here.

Comment (jaredpalmer) [inline packages/backpack-core/config/webpack.config.js]: @sompylasar feel free to submit a PR and I will release and bump version

Comment (sompylasar) [inline packages/backpack-core/config/webpack.config.js]: https://github.com/palmerhq/backpack/pull/45

Comment (JacopKane) [reply packages/backpack-core/config/webpack.config.js]: This banner is writing absolute paths on bundles, and it's hard to notice unless you are using some `CI` environment

Comment (JacopKane) [reply packages/backpack-core/config/webpack.config.js]: created a issue here https://github.com/jaredpalmer/backpack/issues/124

**Pull request merged**